src/bcrhp/bcrhp/templates/views/report-templates/bcrhp_default.htm

```diff
@@ -57,8 +57,59 @@ <h4 class="report-toolbar-title"><span class="bc-report-title" data-bind="text:
     </div>
 </div>
 
-<div class="rp-report-container-preview">
-    <div class="rp-report-section relative rp-report-section-root">
+<div class="rp-report-container-preview relative">
+    <aside id="card-help-panel" class="card-help-panel" style="display: none;" data-bind="visible: helpenable">
+        <div class="relative">
+            <a id="add-basemap-wizard-help-close" href="#" class="help-close fa fa-times fa-lg" style="" data-bind="click: function () { helpactive(false) }"></a>
+        </div>
+        <div id="add-basemap-wizard-help-content">
+            <div>
+                <div class="panel-heading">
+                    <h3 class="panel-title help-panel-title" style="">
+                        <span>Details Card Help Text</span>
+                    </h3>
+                </div>
+                <div class="panel-body" style="padding: 10px 10px 15px 10px;">
+                    <span style="font-weight:bold">Images:</span> Use the left &lt; and right &gt; arrows to view the images.
+                    <br /><br />
+                    <span style="font-weight:bold">Site Name:</span> Includes names associated with the historic place.
+                    <br /><br />
+                    <span style="font-weight:bold">Heritage Site Location:</span> Provides the civic address(es) or location description for the historic place.
+                    <br /><br />
+                    <span style="font-weight:bold">Site ID Number:</span> The Borden System number assigned when the record was created. The Borden System is a Canada-wide map index system to uniquely identify heritage and archaeological sites.
+                    <br /><br />
+                    <span style="font-weight:bold">Official Recognition Information:</span> Describes the status of the historic place on the Register and its associated Registry types. 
+                    <br /><br />
+                        &emsp;&emsp;<span style="font-weight:bold">Registered</span> – the heritage site is officially recognized by a legislative enactment from the province or a municipality/regional district.
```

Comment:
Please make this a block indent. Also, although it was indicated to be a hyphen in the example content, please make it a colon to be consistent with the rest of the items.